extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -517,64 +517,94 @@ export class AmpPanZoom extends AMP.BaseElement {
       }
     });
 
-    this.gestures_.onGesture(PinchRecognizer, e => {
-      const {
-        centerClientX,
-        centerClientY,
-        deltaX,
-        deltaY,
-        dir,
-        last,
-      } = e.data;
-
-      this.onPinchZoom_(centerClientX, centerClientY, deltaX, deltaY, dir);
-      if (last) {
-        this.onZoomRelease_();
-      }
-    });
+    this.gestures_.onGesture(PinchRecognizer,
+        this.handlePinch.bind(this));
```

Comment:
Better to just use an arrow function here (https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions).

Reply:
Done